### PR TITLE
TFJob Common fix links

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/kubeflow/website/issues/2731
/assign @RFMVasconcelos

### CODE REVIEW

File: content/en/docs/reference/tfjob/v1beta2/common.md [hunk 1]
```diff
@@ -59,9 +59,7 @@ JobConditionType
 <td>
 <code>status</code></br>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#conditionstatus-v1-core">
-Kubernetes core/v1.ConditionStatus
-</a>
+string
```
Comment (RFMVasconcelos) [inline content/en/docs/reference/tfjob/v1beta2/common.md:62]: @Arhell is this `string` on purpose?

**Review by RFMVasconcelos — COMMENTED** [review]
Thank you for updating these links @Arhell !! 

Could you please just confirm the string question?

File: content/en/docs/reference/tfjob/v1beta2/common.md (hunk 1 above)
Comment (Arhell) [reply content/en/docs/reference/tfjob/v1beta2/common.md]: @RFMVasconcelos
you are right, this is what I found, but the link is not working unfortunately here too
`status`
https://aws.github.io/aws-app-mesh-controller-for-k8s/reference/api_spec/#appmesh.k8s.aws/v1beta2.GatewayRouteCondition

**Comment by Arhell** [comment]
сс @RFMVasconcelos

**Comment by Arhell** [comment]
> Awesome! is this ready to merge @Arhell ?
> 
> It still appears as draft

@RFMVasconcelos done

**Review by terrytangyuan — APPROVED** [review]
/lgtm

**Pull request merged**